core/field_checkbox.js

```diff
@@ -82,15 +82,6 @@ Blockly.FieldCheckbox.prototype.SERIALIZABLE = true;
  */
 Blockly.FieldCheckbox.prototype.CURSOR = 'default';
 
-/**
- * Used to tell if the field needs to be rendered the next time the block is
- * rendered. Checkbox fields are statically sized, and only need to be
- * rendered at initialization.
- * @type {boolean}
- * @protected
- */
-Blockly.FieldCheckbox.prototype.isDirty_ = false;
```

Comment:
You removed isDirty but not any uses of it.

Reply:
``isDirty_`` is used by field.js to determine whether or not to re-render. Initializing a field with isDirty_ = false was a way for us to mark certain fields "static fields", ie: only init gets called for those fields and not render_.

Removing this because the checkbox field needs to render every time, as the font may have changed and the checkmark is a ``blocklyText`` element which can be resized with the font size.